nixos/modules/services/audio/mpd.nix

```diff
@@ -83,7 +83,7 @@ in {
 
         listenAddress = mkOption {
           type = types.str;
-          default = "any";
+          default = "127.0.0.1";
           description = ''
             This setting sets the address for the daemon to listen on. Careful attention
             should be paid if this is assigned to anything other then the default, any.
```

Comment:
The description still refers to the old default. IMO, it would benefit from some trimming, regardless of this PR (especially the "careful attention" part; I, at least, fail to get anything useful out of it).